src/main/java/frc/robot/RobotContainer.java

```diff
@@ -44,75 +46,144 @@ public class RobotContainer {
   private final Joystick operatorController = new Joystick(Constants.OPERATOR_CONTROLLER);
   
   private final JoystickButton driverControllerBackButton = new JoystickButton(driverController, Constants.BACK_BUTTON);
+  private final JoystickButton driverControllerBButton = new JoystickButton(driverController, Constants.B_BUTTON);
   private final JoystickButton driverControllerYButton = new JoystickButton(driverController, Constants.Y_BUTTON);
   private final JoystickButton driverControllerAButton = new JoystickButton(driverController, Constants.A_BUTTON);
+  private final JoystickButton driverControllerXButton = new JoystickButton(driverController, Constants.X_BUTTON);
+  private final JoystickButton driverControllerLeftBumper = new JoystickButton(driverController, Constants.LEFT_BUMPER);
+  private final JoystickButton driverControllerRightBumper = new JoystickButton(driverController, Constants.RIGHT_BUMPER);
+
+  private final DPadButton driverDPadUp = new DPadButton(operatorController, DPadButton.Direction.UP);
+  private final DPadButton driverDPadLeft = new DPadButton(operatorController, DPadButton.Direction.LEFT);
+  private final DPadButton driverDPadDown = new DPadButton(operatorController, DPadButton.Direction.DOWN);
 
   private final JoystickButton operatorControllerBButton = new JoystickButton(operatorController, Constants.B_BUTTON);
   private final JoystickButton operatorControllerYButton = new JoystickButton(operatorController, Constants.Y_BUTTON);
   private final JoystickButton operatorControllerAButton = new JoystickButton(operatorController, Constants.A_BUTTON);
   private final JoystickButton operatorControllerXButton = new JoystickButton(operatorController, Constants.X_BUTTON);
-  private final JoystickButton operatorControllerLeftBumper = new JoystickButton(operatorController, Constants.LEFT_BUMPER);
-  private final JoystickButton operatorControllerRightBumper = new JoystickButton(operatorController, Constants.RIGHT_BUMPER);
-  private final JoystickButton operatorControllerLeftJoystickButton = new JoystickButton(operatorController, Constants.LEFT_JOYSTICK_BUTTON);
 
   private final DPadButton operatorDPadUp = new DPadButton(operatorController, DPadButton.Direction.UP);
   private final DPadButton operatorDPadRight = new DPadButton(operatorController, DPadButton.Direction.RIGHT);
+  private final DPadButton operatorDPadLeft = new DPadButton(operatorController, DPadButton.Direction.LEFT);
   private final DPadButton operatorDPadDown = new DPadButton(operatorController, DPadButton.Direction.DOWN);
 
   public RobotContainer() {
     configureBindings();
   }
 
+  private boolean armIsInsideRobot() {
+    return arm.getTargetPosition() == Constants.ARM_INSIDE_ROBOT;
+  }
+
+  private boolean floorArmIsInsideRobot() {
+    return floorArm.getTargetPosition() == Constants.FLOOR_ARM_INSIDE_ROBOT;
+  }
```

Comment:
I think the command logic is easier to read if these are instead armIsExtended or floorArmIsExtended